src/game/objects/rabbit.ts

```diff
@@ -52,16 +60,17 @@ export class RabbitCharacter extends Character {
         }
 
         if (this.isSkillActive) {
-            this.eventBlock = 1 / updateInterval
+            this.eventBlock = this.skillPreparationTime
```

Comment:
```suggestion
            this.eventBlock = this.skillPreparationTime
```


항상 eventBlock 이 초기화되어서 if문을 안탈 것 같아요, useSkill에서 eventBlock을 초기화해야하지 않을까요?
그리고 super.update에서는 사전에 감소시키고있네요 🤔 확인 부탁드립니다!